### ISSUE TITLE
ghi doesn't check if issues are enabled

### ISSUE THREAD

**Issue by alfredopalhares** [jsmits/github-cli#52]
Happend on my fork. I was creating some issues and i they didn't show on the project page, but ghi saw them fine.
I had to enable issues on the admin settings. 

It would be nice to have a warning on this. 

I dont know is this is a APIv2 problem tough.

**Comment by jtriley** [comment]
Did the issues you created before enabling in the admin show up on github after enabling or was ghi just creating them locally?

**Comment by alfredopalhares** [comment]
They were really uploaded, I checked from ghi in another machine and only after that i noticed there was no bugs in the web interface
